### PR TITLE
add pagination to global component

### PR DESCRIPTION
**Pull request by matteo-gildone**
Move the `springer-pagination` into `global `

### CODE REVIEW

File: toolkits/global/packages/global-pagination/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Example
+
+#### HTML
+```html
+<nav>
+    <ul class="c-pagination">
+        <li class="c-pagination__item">
+            <span class="c-pagination__link c-pagination__link--disabled">
+                <svg class="c-icon u-margin-right-xs c-pagination__icon" aria-hidden="true">
```
Comment (alexkilgour) [inline toolkits/global/packages/global-pagination/README.md:17]: remove `u-margin-right-xs`. This may not be available and is a product-specific decision

Comment (matteo-gildone) [reply toolkits/global/packages/global-pagination/README.md]: Done!

File: toolkits/global/packages/global-pagination/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Global Pagination
+
+A component for the controls of pagination.
+
+## Branding
+
+To include `global-pagination` component currently uses the `DEFAULT` brand only.
```
Comment (alexkilgour) [inline toolkits/global/packages/global-pagination/README.md:7]: Can you update this section to give an example - along the lines of this
https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-banner#branding

Comment (matteo-gildone) [reply toolkits/global/packages/global-pagination/README.md:7]: Done!

**Pull request merged**